repo2docker/buildpacks/docker.py

```diff
@@ -18,7 +19,15 @@ def render(self, build_args=None):
         """Render the Dockerfile using by reading it from the source repo"""
         Dockerfile = self.binder_path("Dockerfile")
         with open(Dockerfile) as f:
-            return f.read()
+            content = f.read()
+
+        if self.appendix:
+            content += "\n"
+            content += self.appendix
+
+        print(content)
```

Comment:
Should this print statement be added here?

Reply:
Ah, removed!